pennylane/return_types.py

```diff
@@ -615,6 +617,12 @@ def circuit_stack(a, b):
         (tensor([ 0.0000,  0.0050, -0.0050,  0.0050, -0.0050]), tensor([ 0.0000, -0.4888,  0.4888,  0.0012, -0.0012])))
     """
 
+    warnings.warn(
+        "The old return system is deprecated, and will be removed along with "
+        "`qml.enable_return()` in v0.33.",
+        UserWarning,
+    )
+
```

Comment:
For this and the one below, can we also have a recommended action? Something like:
> Please consult the QNode returns page for guidance on switching to the new return system: https://docs.pennylane.ai/en/stable/introduction/returns.html